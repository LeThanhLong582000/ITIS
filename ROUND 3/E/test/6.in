41
5749 4325
6311 8085
865 7823
4651 3345
2977 641
5841 6081
7878 7593
4341 5921
401 7006
971 8601
6241 4961
9202 4489
2481 8084
6033 8881
9404 8211
7301 9951
2481 7885
7761 791
345 4465
4221 5601
6173 5711
713 4485
1956 1601
9275 2601
1577 6401
961 9681
7881 1
5297 785
5961 6281
3771 7381
3417 8313
4165 9361
3328 8769
7987 9136
8993 8524
3884 9009
4206 4609
2131 8736
7951 5149
6621 1048
7121 8841
