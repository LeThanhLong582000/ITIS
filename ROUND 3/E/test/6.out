618239406921672
2603492508022046
45790807143210
242038622261606
3641427647136
1261607637083600
3578151191890466
660647113565300
7892736877050
69748663118750
958620457963200
1706332151691558
402259161030344
2870716581506624
5962347642732704
5278348921703750
382698468469436
37686664232450
2372895374504
558935487537500
1242844410380574
10225928100140
9806614839800
581980427299646
101896356264456
86553930837200
62102280
17290132490688
1401831214341200
774717444360050
806873367772496
1520107850024436
851660767461884
5324508162152196
5876192545997424
1224366687030032
375795652422794
346570777267700
1676058585910494
48146994103604
3963553189191800
